fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/OCFLPersistentStorageUtils.java

```diff
@@ -173,7 +173,9 @@ public static WriteOutcome writeRDF(final OCFLObjectSession session, final RdfSt
         try (final var os = new ByteArrayOutputStream()) {
             final StreamRDF streamRDF = getWriterStream(os, getRdfFormat());
             streamRDF.start();
-            triples.forEach(streamRDF::triple);
+            if(triples != null) {
```

Comment:
space.

Is this covering the case where a client provides an empty body for a rdf source?

Reply:
I believe so, but I'll check.

Reply:
This does not cover the case of an empty body: https://jira.lyrasis.org/browse/FCREPO-3168

Reply:
Okay, what scenario do you get an null `triples` stream in a call to writeRDF?

Reply:
It looks like CreateRdfSourceOperationBuilder passes a null tripleStream when the stream from the request is empty.  So the resulting CreateRdfSourceOperation passes the null triples along.